docs/examples/asyncagi/example.php

```diff
@@ -37,8 +37,7 @@
     implode(
```

Comment:
Well.. how would you specify the include_path before including autoload.php? Command line arguments? Relative paths? I'd rather have this in the example, it clearly states what should be included and from where.